attribution_scopes.md

```diff
@@ -105,68 +105,83 @@ If the current trigger passes the top-level filter check during the attribution
+to select the advertisers:
 
 ```jsonc
 // source registration 1 for advertiser1 at t=0
 {
   ..., // existing fields
-  "attribution_scopes": {
-    "limit": 3,
-    "values": ["advertiser1"],
-    "max_event_states": 3
+  "filter_data": {
+    "advertiser_id": ["advertiser1"]
+  }
+}
+```
+
+``jsonc
+// source registration 2 for advertiser2 at t=1
+{
+  ..., // existing fields
+  "filter_data": {
+    "advertiser_id": ["advertiser2"]
   }
 }
 ```
 
 ```jsonc
-// source registration 2 for advertiser1 at t=1
+// trigger registration 1 for adveriser1 at t=2
 {
   ..., // existing fields
-  "attribution_scopes": {
-    "limit": 3,
-    "values": ["advertiser1"],
-    "max_event_states": 3
+  "filters": {
+    "advertiser_id": ["advertiser1"]
   }
 }
 ```
 
+In this example both source registrations would be eligible for attribution
+based on their reporting origin and destination site, but the API would choose
+the second source registration because it was the most recent source registration.
+However, because the top-level filters (i.e. `filter_data` of the second source
+registration and the `filters` of the trigger) do not match, the API caller would
+not receive an attribution report.
+
+However, if the API caller uses attribution scopes:
+
 ```jsonc
-// source registration 3 for advertiser2 at t=2
+// source registration 1 for advertiser1 at t=0
 {
   ..., // existing fields
   "attribution_scopes": {
-    "limit": 3,
-    "values": ["advertiser2"],
-    "max_event_states": 3
+    "limit": 2,
+    "values": ["advertiser1"],
   }
 }
 ```
+
 ```jsonc
-// source registration 4 for advertiser3 at t=3
+// source registration 2 for advertiser2 at t=1
 {
   ..., // existing fields
   "attribution_scopes": {
-    "limit": 3,
-    "values": ["advertiser3"],
-    "max_event_states": 3
+    "limit": 2,
+    "values": ["advertiser2"],
   }
 }
 ```
 
-The user then converts at a later time on the destination site by purchasing a product associated with advertiser1:
-
 ```jsonc
-// trigger registration 1 for advertiser1 at t=4
+// trigger registration 1 for advertiser1 at t=2
 {
   ..., // existing fields
   "attribution_scopes": ["advertiser1"]
 }
 ```
 
-The API automatically performs attribution between any sources that have `attribution_scopes/values` that are not disjoint with the trigger `attribution_scopes`. Any sources that do not have an `attribution_scopes/values` that matches at least one of the trigger registration `attribution_scopes` are deleted (assuming the source that is chosen passes the top-level filter check; if it does not then no sources are deleted). In this example, the API caller would receive an attribution report attributing the trigger registration to advertiser1’s second source registration.
+The API automatically performs attribution between any sources that have `attribution_scopes/values` that are not disjoint with the trigger `attribution_scopes`.
```

Comment:
```suggestion
The API automatically performs attribution between any sources that have `attribution_scopes/values` that are not disjoint with the trigger `attribution_scopes`.
```
```suggestion
The API starts by checking to see if there are any sources that have `attribution_scopes/values` that are not disjoint with the trigger `attribution_scopes. The API will then use that set of sources to perform the rest of the attribution process. In this example, the API would start by selecting the first source registration since it is the only source that has a not disjoint set with the trigger `attribution_scopes`.  After this has been checked the API will continue with the rest of the attribution checks.
```